fixture-monkey-api/src/main/java/com/navercorp/fixturemonkey/api/random/RegexGenerator.java

```diff
@@ -49,6 +63,38 @@ public final class RegexGenerator {
 		PREDEFINED_CHARACTER_CLASSES = Collections.unmodifiableMap(characterClasses);
 	}
 
+	public String generate(String regex, int[] flags, @Nullable Integer min, @Nullable Integer max, boolean notBlank) {
```

Comment:
Since it is a utility method, its parameters should not be too detailed.
It is better to replace `min`, `max`, `notBlank` with `Predicate<String>`

Reply:
Using Predicate<String> seems much better! Thanks for the review.